packages/core/src/services/documents/destroyOrSoftDeleteDocuments.test.ts

```diff
@@ -0,0 +1,87 @@
+import { database } from '$core/client'
+import { documentVersions } from '$core/schema'
+import { updateDocument } from '$core/services/documents/update'
+import * as factories from '$core/tests/factories'
+import { and, eq } from 'drizzle-orm'
+import { describe, expect, it } from 'vitest'
+
+import { destroyOrSoftDeleteDocuments } from './destroyOrSoftDeleteDocuments'
+
+describe('destroyOrSoftDeleteDocuments', () => {
+  it('hardDestroyDocuments', async () => {
```

Comment:
can we make the titles self explanatory? why does it hard destroy in this test case but it creates in the next test case? It would be awesome if i could know the rule by just reading this line